### PR TITLE
Update Sentry instructions

### PR DESCRIPTION
**Pull request by macav**

### CODE REVIEW

File: templates/config/initializers/sentry.rb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
-if defined?(Raven)
-  Raven.configure do |config|
-    config.processors -= [Raven::Processor::PostData]
```
Comment (macav) [inline templates/config/initializers/sentry.rb:5]: This is no longer supported, nor needed: https://docs.sentry.io/platforms/ruby/migration/#removed-processors

File: templates/config/initializers/sentry.rb [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+if defined? Sentry
+  Sentry.init do |config|
+    filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters)
+    config.before_send = lambda do |event, _hint|
+      filter.filter(event.to_hash)
+    end
```
Comment (macav) [inline templates/config/initializers/sentry.rb:8]: I propose to add the parameter filtering as a default (https://docs.sentry.io/platforms/ruby/guides/rails/configuration/options/#optional-settings)

The `filter_parameter` config is defined by default in new projects in `config/initializers/filter_parameter_logging.rb`

**Review by schmijos — APPROVED** [review]
What about the environments?

**Comment by macav** [comment]
@schmijos the environments were already documented before (using ENV variable `SENTRY_ENVIRONMENT`). Nothing changes there: https://github.com/renuo/applications-setup-guide/blob/master/ruby_on_rails/sentry.md#backend-rails

**Pull request merged**